### PR TITLE
[SCHEMA] schematization of bep029

### PR DESCRIPTION
**Pull request by rwblair**
This is a work in progress. I didn't want to pollute bep029 PR with schema related things until it was ready. Unfortunately I forked off master and not the bep029 branch.

### CODE REVIEW

File: src/schema/objects/entities.yaml [hunk 1]
```diff
@@ -377,3 +377,11 @@ tracer:
...
   type: string
   format: label
+tracking_system:
+    name: tracksys
+    display_name: Tracking System
+    description: |
+      A tracking system is defined as a group of tracking devices that share hardware properties
+      (same recording device) and/or software properties (for example same sampling rates).
+    type: string
+    fromat: label
```
Comment (tsalo) [inline src/schema/objects/entities.yaml:387]:
```suggestion
    format: label
```

File: src/schema/objects/columns.yaml [hunk 2]
```diff
@@ -52,6 +52,18 @@ color:
...
+    display_name: Coordinate System Component
+    description: |
+        Component of the representational system described in `*_coordinatesystem.tsv` that the
+        channel contains. Must be one of: "x" ,"y", "z", "n/a"
+    type: string
+    enum:
+    - "x"
+    - "y"
+    - "z"
+    - "n/a"
```
Comment (bendhouseart) [inline src/schema/objects/columns.yaml:66]:
```suggestion
component:
  name: component
  display_name: Coordinate System Component
  description: |
    Component of the representational system described in `*_coordinatesystem.tsv` that the
    channel contains. Must be one of: "x" ,"y", "z", "n/a"
  type: string
  enum:
  - "x"
  - "y"
  - "z"
  - "n/a"
```

File: src/schema/objects/entities.yaml (hunk 1 above)
Comment (bendhouseart) [inline src/schema/objects/entities.yaml:387]:
```suggestion
tracking_system:
  name: tracksys
  display_name: Tracking System
  description: |
    A tracking system is defined as a group of tracking devices that share hardware properties
      (same recording device) and/or software properties (for example same sampling rates).
  type: string
  format: label
```

File: src/schema/objects/metadata.yaml [hunk 3]
```diff
@@ -2481,6 +2481,32 @@ Resolution:
...
+    A 3D rotation is represented about 3 different axes (Tait-Bryan convention). Must be one of:
+    "XYZ", "XZY", "YXZ", "YZX", "ZXY", "ZYX".
+  type: string
+  enum:
+  - XYZ
+  - XZY
+  - YXZ
+  - YZX
+  - ZXY
+  - ZYX
```
Comment (bendhouseart) [inline src/schema/objects/metadata.yaml:2498]:
```suggestion
  - ZYX
```

File: src/schema/objects/metadata.yaml [hunk 4]
```diff
@@ -2481,6 +2481,32 @@ Resolution:
...
+  - YZX
+  - ZXY
+  - ZYX 
+RotationRule:
+  name: RotationRule
+  display_name: Rotation Rule
+  description: |
+    In case orientation channels are present, indicate whether rotations are applied
+    clockwise around an axis when seen from the positive direction (left-hand rule) or
+    counter-clockwise (right-hand rule). Must be one of: "left-hand", "right-hand".
```
Comment (bendhouseart) [inline src/schema/objects/metadata.yaml:2505]:
```suggestion
    counter-clockwise (right-hand rule). Must be one of: "left-hand", "right-hand".
```

File: src/schema/objects/metadata.yaml [hunk 5]
```diff
@@ -2481,6 +2481,32 @@ Resolution:
   - type: object
     additionalProperties:
       type: string
+RotationOrder:
+  name: RotationOrder
+  display_name: RotationOrder
+  description: |
+    Specify the sequence in which the elemental rotations are applied around the extrinsic axes.
+    A 3D rotation is represented about 3 different axes (Tait-Bryan convention). Must be one of:
+    "XYZ", "XZY", "YXZ", "YZX", "ZXY", "ZYX".
```
Comment (tsalo) [inline src/schema/objects/metadata.yaml:2490]:
```suggestion
    A 3D rotation is represented about 3 different axes (Tait-Bryan convention).
```
We don't need to list enums in the descriptions.

**Pull request closed without merging**